linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_bashrc_exec_tmux/rule.yml

```diff
@@ -26,8 +26,12 @@ ocil_clause: 'exec tmux is not present at the end of bashrc'
 ocil: |-
     To verify that tmux is configured to execute,
     run the following command:
-    <pre>$ tail -1 /etc/bashrc</pre>
+    <pre>$ tail -5 /etc/bashrc</pre>
```

Comment:
I wanted the full snippet to be visible, so that the user can see / double check it. But yes, this indeed can be done by grep:
```
$ grep -A1 -B3 "case ..name. in sshd|login) exec tmux ;; esac" /etc/bashrc
if [ "$PS1" ]; then
  parent=$(ps -o ppid= -p $$)
  name=$(ps -o comm= -p $parent)
  case "$name" in sshd|login) exec tmux ;; esac
fi
```
Will update the commit.